app/retail/templates/shared/footer_scripts_ligth.html

```diff
@@ -0,0 +1,87 @@
+{% comment %}
+    Copyright (C) 2019 Gitcoin Core
+
```

Comment:
Typo in `footer_scripts_ligth.html` -> `footer_scripts_lite` / `footer_scripts_light` ?

Reply:
ohhhh good catch

Reply:
I will use lite